VultisigApp/VultisigApp/Chains/OneInchSwaps.swift

```diff
@@ -58,14 +61,15 @@ private extension OneInchSwaps {
                 }
             }
         }
-
-        let gasPrice = BigUInt(quote.tx.gasPrice) ?? BigUInt.zero
-        // sometimes the `gas` field in oneinch tx is 0
-        // when it is 0, we need to override it with defaultETHSwapGasUnit(600000)
+        
+        guard case .Ethereum(let maxFeePerGasWei, _, _, _) = keysignPayload.chainSpecific else {
```

Comment:
Can you please don't change this one , it will break the 1inch swap cross device.